taliforth.asm

```diff
@@ -111,16 +142,61 @@ dodefer:
 
                 jmp (tmp2)      ; This is actually a jump to the new target
 
-defer_error:
-                ; """Error routine for undefined DEFER: Complain and abort"""
-                lda #err_defer
-                jmp error
+
+domarker:
+                ; keep actual code next to MARKER since save/restore mirror each other
+                jmp marker_runtime
+
+known_cfa_end:
+
+; dodoes is special because it appears as the target of dynamic CFA jsr
 
 dodoes:
         ; """Execute the runtime portion of DOES>. See DOES> and
         ; docs/create-does.txt for details and
         ; http://www.bradrodriguez.com/papers/moving3.htm
         ; """
+                ; typically called like
```

Comment:
I slowly wrote this comment for my own understanding of how this works, whether it's useful for others I don't know.